api-scanner/src/test/java/net/corda/plugins/KotlinAutoGeneratedClassTest.java

```diff
@@ -0,0 +1,30 @@
+package net.corda.plugins;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TemporaryFolder;
+import org.junit.rules.TestRule;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+
+public class KotlinAutoGeneratedClassTest {
+    private final TemporaryFolder testProjectDir = new TemporaryFolder();
+    private final GradleProject testProject = new GradleProject(testProjectDir, "kotlin-auto-generated-class");
+
+    @Rule
+    public TestRule rules = RuleChain.outerRule(testProjectDir).around(testProject);
+
+    @Test
+    public void testKotlinAutoGeneratedClassMethod() throws IOException {
+        System.out.println("Started output");
+        System.out.println(CopyUtils.toString(testProject.getApi()));
+        System.out.println("Finished output");
+        assertEquals("public final class net.corda.example.ContainsAutoGeneratedClass extends java.lang.Object\n" +
```

Comment:
Did you mean to leave those `System.out.println()` lines in?

Reply:
Nope, I've deleted